app/views/categories/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1 class="mb-3">Categories</h1>
+
+<% @categories.each do |category| %>
+  <div class="card mb-1">
+    <div class="card-body">
+      <div class="row">
+        <div class="col">
+          <%= image_tag(category.icon, alt: "Icon pic", height: "40px") %>
+        </div>
+        <div class="col">
+          <%= link_to category.name, category_path(category) %>
+        </div>
+        <div class="col">
+          $<%= category.expenses.sum(:amount) %>
+        </div>
+      </div>
+    </div>
+  </div>
+<% end %>
+
+<%= link_to "Add new category", new_category_path, class: "btn btn-success mt-3" %>
```

Comment:
Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_532](https://user-images.githubusercontent.com/69638013/202434580-f41c1cd3-b285-4ff3-8c10-83e1d9ee9def.png) |  ![Screenshot_533](https://user-images.githubusercontent.com/69638013/202434536-d86bf07b-32bb-482f-9f4d-a3bfaae114b0.png)

- [ ] It would be better to add the Navigation header to your application and move the `add new category`  button to the bottom, this will make your app look like the design, please don't forget to personalize the header content

Reply:
Noted.